engine/servermaster/campaign.go

```diff
@@ -192,19 +177,13 @@ func (s *Server) checkLeaderExists(ctx context.Context) (retry bool, err error)
 		}
 
 		// step-3, case-c
-		leader.IsServLeader = true
-		leader.IsEtcdLeader = true
+		leader.IsLeader = true
 		log.Info("start to watch server master leader",
 			zap.String("leader-name", leader.Name), zap.String("addr", leader.AdvertiseAddr))
-		s.watchLeader(ctx, leader, rev)
+		s.watchLeader(ctx, leader, key, rev)
 		log.Info("server master leader changed")
 	}
 
-	// step-3
```

Comment:
Can remove `step-3` in the comment of function `checkLeaderExists`

Reply:
Done.